syntax = "proto2";
import "session_event.proto";

message Event_NotifyUser {

    enum NotificationType {
        UNKNOWN = 0;  // Default enum value if no "type" is defined when used
        PROMOTED = 1;
        WARNING = 2;
        IDLEWARNING = 3;
    }

    extend SessionEvent {
        optional Event_NotifyUser ext = 1010;
    }
    optional NotificationType type = 1;
    optional string warning_reason = 2;

}
